docs/core/install/linux-package-mixup.md

```diff
@@ -50,25 +50,25 @@ Mixing .NET packages from two different sources will most likely lead to issues
 
 The solution to these problems is to use .NET from one package repository. Which repository to pick, and how to do it, varies by use-case and the Linux distribution.
 
-If your distribution provides .NET packages, it's recommended that you use that package repository instead of Microsoft's.
+If your distribution provides .NET packages, and you only need those packages, it's recommended that you use that package repository instead of Microsoft's.
 
 01. **I only use .NET and no other packages from the Microsoft repository, and my distribution provides .NET packages.**
 
     If you only use the Microsoft repository for .NET packages and not for any other Microsoft package such as `mdatp`, `powershell`, or `mssql`, then:
 
-    01. Remove the Microsoft repository.
     01. Remove the .NET related packages from your OS.
+    01. Remove the Microsoft repository.
     01. Install the .NET packages from the distribution repository.
 
     For Fedora, CentOS 8+, RHEL 8+, use the following bash commands:
 
     ```bash
-    sudo dnf remove packages-microsoft-prod
     sudo dnf remove 'dotnet*' 'aspnet*' 'netstandard*'
+    sudo dnf remove packages-microsoft-prod
```

Comment:
To confirm the Microsoft repository was removed, use the following command:

```bash
cat /etc/yum.repos.d/microsoft-prod.repo
```

If the file is not present, the command will return 

```bash
cat: /etc/yum.repos.d/microsoft-prod.repo: No such file or directory
```